salt/scripts.py

```diff
@@ -588,14 +588,40 @@ def salt_unity():
     s_fun()
 
 
+def _pip_args(args, target):
+    new_args = args[:]
+    target_in_args = False
+    for arg in args:
+        if "--target" in arg:
+            target_in_args = True
+    if "install" in args and not target_in_args:
+        new_args.append(f"--target={target}")
+    return new_args
+
+
+def _pip_environment(env, extras):
+    new_env = env.copy()
+    if "PYTHONPATH" in env:
+        sep = ":"
+        if sys.platform == "win32":
+            sep = ";"
+        new_env["PYTHONPATH"] = f"{extras}{sep}{env['PYTHONPATH']}"
+    else:
+        new_env["PYTHONPATH"] = extras
+    return new_env
+
+
 def salt_pip():
     """
     Proxy to current python's pip
     """
+    extras = str(sys.RELENV / "extras-{}.{}".format(*sys.version_info[:2]))
```

Comment:
```suggestion
    extras = str(sys.RELENV / "extras-{}.{}".format(*sys.version_info))
```
No slicing is actually required